onnxruntime/test/unittest_main/test_main.cc

```diff
@@ -54,8 +57,16 @@ int TEST_MAIN(int argc, char** argv) {
 
   ORT_TRY {
     ::testing::InitGoogleTest(&argc, argv);
-
     ortenv_setup();
+
+    // allow verbose logging to be enabled by setting this environment variable to 1
+    constexpr auto kEnableVerboseLoggingEnvironmentVariableName = "ORT_UNIT_TEST_MAIN_ENABLE_VERBOSE_LOGGING";
```

Comment:
Could we make it more generic and use whatever value the env var has as the logging level? 

e.g ORT_UNIT_TEST_MAIN_LOG_LEVEL=0 is verbose, =1 is info etc.